Cataclysm-DDA.sublime-project

```diff
@@ -0,0 +1,98 @@
+{
+	"build_systems":
+	[
+		{
+			"cmd":
+			[
+				"make",
+				"USE_HOME_DIR=1",
+				"CLANG=1",
+				"CCACHE=1",
+				"RELEASE=1",
+				"LUA=1"
+			],
+			"file_regex": "^(..[^:]*):([0-9]+):?([0-9]+)?:? (.*)$",
+			"name": "make RELEASE LUA",
+			"working_dir": "${folder:${project_path:${file_path}}}"
+		},
+		{
+			"cmd":
+			[
+				"make",
+				"USE_HOME_DIR=1",
+				"CLANG=1",
+				"CCACHE=1",
+				"LUA=1"
+			],
+			"file_regex": "^(..[^:]*):([0-9]+):?([0-9]+)?:? (.*)$",
+			"name": "make LUA",
+			"working_dir": "${folder:${project_path:${file_path}}}"
+		},
+		{
+			"cmd":
+			[
+				"make",
+				"USE_HOME_DIR=1",
+				"CLANG=1",
+				"CCACHE=1",
+				"LUA=1",
+				"TILES=1"
+			],
+			"file_regex": "^(..[^:]*):([0-9]+):?([0-9]+)?:? (.*)$",
+			"name": "make TILES LUA",
+			"working_dir": "${folder:${project_path:${file_path}}}"
+		},
+		{
+			"cmd":
+			[
+				"make",
+				"USE_HOME_DIR=1",
+				"CLANG=1",
+				"CCACHE=1",
+				"LUA=1",
+				"TILES=1",
+				"RELEASE=1"
+			],
+			"file_regex": "^(..[^:]*):([0-9]+):?([0-9]+)?:? (.*)$",
+			"name": "make RELEASE TILES LUA",
+			"working_dir": "${folder:${project_path:${file_path}}}"
+		}
+	],
+	"folders":
+	[
+		{
+			"path": "."
+		},
+		{
+			"path": "/home/sparr/.cataclysm-dda"
```

Comment:
I swear that had a tilde in it when I wrote it. It got expanded somewhere along the way.